rpcs3/rpcs3qt/debugger_frame.cpp

```diff
@@ -814,6 +828,11 @@ void debugger_frame::UpdateUnitList()
 		if (m_inst_editor) m_inst_editor->close();
 	}
 
+	if (emu_state == system_state::stopped)
+	{
+		ClearBreakpoints();
```

Comment:
Im a bit confused. I thought you guys wanted to keep breakpoints when working on the same game for a while.

Reply:
It's super broken currently, may be implmented in the future.